compiler/rustc_codegen_ssa/src/traits/backend.rs

```diff
@@ -54,6 +55,12 @@ pub trait CodegenBackend {
     fn print_passes(&self) {}
     fn print_version(&self) {}
 
+    /// If this plugin provides additional builtin targets, provide the one enabled by the options here.
+    /// Be careful: this is called *before* init() is called.
```

Comment:
Does the llvm codegen backend already check that `init` is called before anything else? If not it would be nice to check just in case. I think there is already a function for this used in some cases.

Reply:
Maybe open a separate issue? This PR doesn't add a `target_override` for the LLVM backend.